x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/add_prepackaged_rules_route.ts

```diff
@@ -46,14 +51,19 @@ export const addPrepackedRulesRoute = (
   router.put(
     {
       path: DETECTION_ENGINE_PREPACKAGED_URL,
-      validate: false,
+      validate: {
+        body: buildRouteValidation<
+          typeof installPrepackagedRulesSchema,
+          InstallPrepackagedRulesSchema
+        >(installPrepackagedRulesSchema),
+      },
```

Comment:
You'll have to mess with these types and figure out how to make the body optional. I am pretty sure the way to do that is to make the type basically a union with null so that when you have `request.body` the body value can be a null value.